src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -991,18 +993,136 @@ The above program will produce the following types for acceleration quantities
 
 ## Scaled units
 
+Another very common operation is to multiply an existing unit by a factor, creating a new, scaled unit.  For
+example, the unit _foot_ can be multiplied by 3, producing the unit _yard_.
+
+The process also works in reverse: the ratio between any two units of the same dimension is a well-defined
+number.  For example, the ratio between one foot and one inch is 12.
+
 ### Unit magnitudes
 
-TBD
+In principle, this scaling factor can be any positive real number.  In mp-units and Au, we have used the term
+"magnitude" to refer to this scaling factor.  (This should not be confused with the logarithmic "magnitude"
+unit commonly used in astronomy.)
 
-### Construction helpers
+In the library implementation, each unit is associated with a magnitude.  However, for most units the
+magnitude is a fully encapsulated implementation detail, not a user-facing value.
 
+This is because the notion of "the" magnitude of a unit is not generally meaningful: it has no physically
+observable consequence.  What _is_ meaningful is the _ratio_ of magnitudes between two units of the same
+dimension.  We could associate the _foot_, say, with any magnitude $m_f$ that we like --- but once we make that
+choice, we must assign $3m_f$ to the _yard_, and $m_f/12$ to the _inch_.  Separately and independently, we can
+assign any magnitude $m_s$ to the second, because it's an independent dimension --- but once we make that
+choice, it fixes the magnitude for derived units, and we must assign, say, $(5280 m_f) / (3600 m_s)$ to the
+_mile per hour_.
```

Comment:
```suggestion
_miles per hour_.
```

Reply:
I think the original is correct here.  "the mile per hour" is grammatically consistent with "the foot" and "the inch" above.